2001-3000/LIT2835RepCh119.xml

```diff
@@ -66,25 +68,66 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                   <ptr target="bm:Chaine1913Rep"/>
                   <citedRange unit="item">119</citedRange>
                </bibl>
+               <bibl><ptr target="bm:MgWebsite"/></bibl>
+               <bibl><ptr target="bm:Malkea2020Am"/></bibl>
+               <bibl><ptr target="bm:MalkaGubae1974"/></bibl>
+               <bibl><ptr target="bm:ZekraAbaw"/></bibl>
             </listBibl>
          </div>
-         <div type="edition">
-      
-            <div type="textpart" subtype="incipit">
-               <ab>መልክአ፡ ሚካኤል። ሰላም፡ ለዝክረ፡ ስምከ፡ ምስለ፡ ስመ፡ ልዑል፡ ዘተሳተፈ፡ ወልደ፡ ያሬድ፡ ሔኖክ፡ በከመ፡ ጸሐፈ፡ ሶበ፡ እጼውዕ፡ ስመከ፡ ከሢትየ፡ አፈ፡ ራዳኤ፡ ምንዱባን፡ ሚካኤል፡ በከመ፡ ታለምድ፡ ዘልፈ።</ab>
```

Comment:
As far as I see, the text is completely different. Are you sure that this is the same work? Should the old incipit and explicit be preserved as supplications or are they redundant?

Reply:
It is not different if you look at stanza 1 and not at the introduction

Reply:
```
<div type="textpart" xml:id="malke">
               <lg type="stanza" n="1">
                  <l n="1">ሰላም፡ ለዝክረ፡ ስምከ፡ ምስለ፡ ስመ፡ ልዑል፡ ዘተሳተፈ፨</l>
                  <l n="2">ወልደ፡ ያሬድ፡ ሄኖክ፡ በከመ፡ ጸሐፈ፨</l>
                  <l n="3">ሶበ፡ እጼውዕ፡ ስመከ፡ ከሢትየ፡ አፈ፨</l>
                  <l n="4">ራዳኤ፡ ምንዱባን፡ ሚካኤል፡ በከመ፡ ታለምድ፡ ዘልፈ፨</l>
                  <l n="5">ለረዲኦትየ፡ ነዓ፡ ሰፊሐከ፡ ክንፈ፨</l>
               </lg>
```

Reply:
`               <ab>መልክአ፡ ሚካኤል። ሰላም፡ ለዝክረ፡ ስምከ፡ ምስለ፡ ስመ፡ ልዑል፡ ዘተሳተፈ፡ ወልደ፡ ያሬድ፡ ሔኖክ፡ በከመ፡ ጸሐፈ፡ ሶበ፡ እጼውዕ፡ ስመከ፡ ከሢትየ፡ አፈ፡ ራዳኤ፡ ምንዱባን፡ ሚካኤል፡ በከመ፡ ታለምድ፡ ዘልፈ።</ab>
`